flink-connectors/flink-sql-connector-hive-3.1.3/pom.xml

```diff
@@ -223,6 +223,10 @@ under the License.
 									<pattern>org.apache.parquet</pattern>
 									<shadedPattern>org.apache.hive.shaded.parquet</shadedPattern>
```

Comment:
Nit) Shading naming convention looks inconsistent with the one in ```flink-sql-connector-2.3.9```.
How about chaing this to ```org.apache.flink.hive.shaded.parquet```?

Reply:
No, I dont' need we need. Otherwise it'll cause some problems.
See https://issues.apache.org/jira/browse/FLINK-23074?focusedCommentId=17374459&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17374459